clients/java/zts/src/main/java/com/yahoo/athenz/zts/AWSCredentialsProviderImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.yahoo.athenz.zts;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.BasicSessionCredentials;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AWSCredentialsProviderImpl implements AWSCredentialsProvider {
+    private static final Logger LOG = LoggerFactory.getLogger(AWSCredentialsProviderImpl.class);
+    private static String athensSvcDomain = "";
+    private static String athensDomRole = "";
+    private volatile AWSCredentials credentials;
+    private ZTSClient ztsClt;
+
+
+    public AWSCredentialsProviderImpl(ZTSClient ztsClt, String athensSvcDomain, String athensDomRole) {
+        this.ztsClt = ztsClt;
+        this.athensDomRole = athensDomRole;
+        this.athensSvcDomain = athensSvcDomain;
+    }
+
+    @Override
+    public AWSCredentials getCredentials() {
+        this.refresh();
+        return this.credentials;
+    }
+
+    @Override
+    public void refresh() {
+        try {
+            long start = System.currentTimeMillis();
```

Comment:
time stamp support was only for debugging and must not be in the code. let's remove any references.